### ISSUE TITLE
Send command

### ISSUE THREAD

**Issue by wagnerdracha** [tananaev/traccar#1471]
Hi,

i'm trying to send a command to Traccar by PHP, but without success.

The return is: stdClass Object ( [success] => [error] => User not logged in )

How can I do?

Thank's

**Comment by tananaev** [comment]
As the error message suggests, you need to login first. See following page for more details:

https://www.traccar.org/traccar-api/

**Comment by mtrntx** [comment]
It's possible that when I try to use the API without login or login with a master user, because I try to used from php but I made some changes into db to login because I made a implementacion of ACL ( Access Control List ) and I don't make a relation with the user and device

**Comment by tananaev** [comment]
No, you need to be logged in as a user that has the device your are trying to send command to in his list.
